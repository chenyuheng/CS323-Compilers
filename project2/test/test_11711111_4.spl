struct node {
    int value;
    struct node next;
};
int main() {
    struct node n;
    struct node n1 = n.next.next;
    int a = n1.next;
    struct node t = n1.previous;
    return a.next;
}
