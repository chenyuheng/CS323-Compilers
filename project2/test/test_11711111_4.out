Error type 5 at Line 8: unmatching type on both sides of assignment
Error type 14 at Line 9: no such member: previous
Error type 5 at Line 9: unmatching type on both sides of assignment
Error type 13 at Line 10: accessing with non-struct variable
